src/routes/note-edit.test.js

```diff
@@ -0,0 +1,76 @@
+import { MemoryRouter } from "react-router-dom";
+import { render, cleanup, act, fireEvent } from '@testing-library/react';
+
+import NoteEditRoute from './note-edit';
+import Harness from '../../test/jest/helpers/harness';
+
+jest.mock('@folio/stripes/smart-components', () => ({
+    NoteEditPage:
+        ({
+            navigateBack,
+        }) => (
```

Comment:
I think, you can leave it on one line here as you have only one variable here.
```suggestion
        ({ navigateBack }) => (
```